pages/apim/3.x/installation-guide/docker/installation-guide-docker_customize.adoc

```diff
@@ -70,3 +70,23 @@ $ docker run  \
         --detach  \
         graviteeio/apim-gateway:latest
 ....
+
+=== Use JDBC as the datastore for management (since 3.5)
+
+The JDBC plugin is part of the default distribution since version 3.5 so this step is a bit different.
+
+NOTE: You must ensure that you have the correct JDBC driver for your RDBMS (the example assumes the JDBC connector is available from `./plugins/ext/repository-jdbc/` on the Docker Host).
+
+The driver has to be in the same directory hierarchy than the JDBC plugin. So the volume must be mounted underneath the default `/plugin/` directory. The `/plugin-ext/` directory from the example above is no longer needed.
```

Comment:
The driver needs to be in the same directory hierarchy as the JDBC plugin, so the volume must be mounted under ...